src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -49,10 +49,8 @@ trait FireCloudDirectives extends spray.routing.Directives with PerRequestCreato
     val path = encodeUri(unencodedPath)
     // POST, PUT, PATCH
     if (inMethod.isEntityAccepted) {
-      method(inMethod) {
-        respondWithJSON { requestContext =>
```

Comment:
I made this change because in some cases rawls and other services return plaintext. Since we force this header onto the response of all passthroughs, when we actually want to pass plaintext through, Swagger-UI will show "no response from server" because the content type doesn't match what's being served.

I'm not sure if this has other implications but I at least wanted to bring it up for discussion